README.md

```diff
@@ -1,8 +1,66 @@
-# Microverse first repo
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-My first repo of microverse
+# Hello Microverse
 
-## Author
-Michael Hirpo
-mikaelhirpo@gmail.com
+> Description the project.
 
+
+## Built With
+
+- Html
+- css
+
+
+
+
+
+## Getting Started
+
+**git clone https://github.com/mikihirpo/microverse-first.git**
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/mikihirpo)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/michael-hirpo-40269882/)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Great job adding a descriptive README, however as the previous reviewer suggested, please add an `MIT.md` file in your root folder. Here is a demo below:

https://user-images.githubusercontent.com/52934128/176409659-c6c77558-b1f5-416c-ab69-401853e303f6.mp4